chart/hyrax/templates/configmap-env.yaml

```diff
@@ -19,15 +19,20 @@ data:
   HYRAX_CACHE_PATH: "/app/samvera/file_cache"
   HYRAX_UPLOAD_PATH: "/app/samvera/uploads"
   {{- end }}
+  {{- if .Values.loadHyraxEngineSeed }}
```

Comment:
GitHub wouldn't let me do a multi-line suggestion, but now this is unneeded i think(?)

Reply:
I think it is, because the check in the dassie seed file does a `ENV[].present?`. So we don't want any
variable defined in that scenario, right?

Reply:
i think we should change that line to use `ActiveModel::Types::Boolean`, which is how we interpret boolean ENV variables elsewhere in the codebase.

that way users can do an explicit `false` or `0`

Reply:
Ok, I'm not familiar with that pattern but I'll look and update once I've got something working using that.

Reply:
see https://api.rubyonrails.org/v6.1.0/classes/ActiveModel/Type/Boolean.html